static/app/components/events/autofix/autofixTimeline.tsx

```diff
@@ -160,4 +160,5 @@ const StyledTimelineHeader = styled('div')<{isActive?: boolean}>`
 const StyledIconChevron = styled(IconChevron)`
   color: ${p => p.theme.gray300};
   flex-shrink: 0;
+  margin-right: ${space(0.25)};
```

Comment:
margin to make sure they're aligned on root cause and solution after solution now has buttons on the right